Assets/Scripts/Strategy/BaseManagement/AbstractProductionBuilding.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class AbstractProductionBuilding : AbstractBuilding
+{
+    public AbstractProductionBuilding() : base() { }
+}
```

Comment:
What is the point of this class?

If it is kept it needs a namespace

Reply:
612bc2f